scalapact-sbtplugin/src/main/scala/com/itv/scalapact/plugin/verifier/ScalaPactVerifyCommand.scala

```diff
@@ -40,4 +41,4 @@ object ScalaPactVerifyCommand {
 }
 
 case class ProviderState(key: String, f: String => Boolean)
-case class PactVerifySettings(providerStates: List[ProviderState], pactBrokerAddress: String, projectVersion: String, providerName: String, consumerNames: List[String])
+case class PactVerifySettings(providerStates: List[ProviderState], pactBrokerAddress: String, projectVersion: String, providerName: String, consumerNames: List[String], versionedConsumerNames: List[(String,String)])
```

Comment:
I'd prefer it if you create a case class so that you got `List[VersionedConsumer]` rather than `List[(String, String)]` but that's personal taste. :-)